FantasmoSDK/app/src/main/java/com/fantasmo/sdk/models/tensorflowML/ImageQualityModelUpdater.kt

```diff
@@ -102,11 +103,11 @@ class ImageQualityModelUpdater(val context: Context) {
         return if (::interpreter.isInitialized) {
             interpreter
         } else {
-            var result = loadFromAssets()
-            if (result == null) {
-                result = loadFromURL()
+            if (!hasTriedAssets){
```

Comment:
It's important that if the remote config returns a higher value for the model **version**, the app downloads the latest model (even if one is present/already loaded into memory). Is that the case?